protocol/contracts/v2/strategies/AbstractStrategy.sol

```diff
@@ -197,16 +193,22 @@ abstract contract AbstractStrategy is ITempleStrategy, TempleElevatedAccess {
     }
 
     /**
-     * @notice Executors can set the allowance of any token spend from the strategy
+     * @dev Set the allowance of any token spend from the strategy
      */
-    function setTokenAllowance(IERC20 token, address spender, uint256 amount) external override onlyElevatedAccess {
+    function _setTokenAllowance(IERC20 token, address spender, uint256 amount) internal {
         if (amount == token.allowance(address(this), spender)) return;
 
-        emit TokenAllowanceSet(address(token), spender, amount);
         token.safeApprove(spender, 0);
-
         if (amount != 0) {
             token.safeIncreaseAllowance(spender, amount);
         }
     }
+
+    /**
+     * @notice Executors can set the allowance of any token spend from the strategy
+     */
```

Comment:
add forge test for external function

Reply:
There already is, no?

https://github.com/TempleDAO/temple/blob/290affd0b0a693be3ffd5a6b9b6986f65b178721/protocol/test/forge/v2/strategies/AbstractStrategy.t.sol#L93